WordPress/Classes/Services/PostCoordinator.swift

```diff
@@ -321,7 +321,8 @@ class PostCoordinator: NSObject {
         assert(isSyncAllowed(for: revision.original()), "Sync is not supported for this post")
 
         if !revision.isSyncNeeded {
-            revision.isSyncNeeded = true
+            revision.remoteStatus = .syncNeeded
+            revision.confirmedChangesTimestamp = Date()
```

Comment:
We're not using this `confirmedChangesTimestamp` yet, but it's an important piece of information.